fr-French/strings.xml

```diff
@@ -1,301 +1,301 @@
      <!-- Sign up -->
-    <string name="sign_up">Sign up</string>
-    <string name="sign_up_for_simplelogin">Sign up for SimpleLogin</string>
-    <string name="sign_up_success">Check your inbox for verification code</string>
-    <string name="minimum_8_characters">Minimum 8 characters</string>
+    <string name="sign_up">S\'inscrire</string>
+    <string name="sign_up_for_simplelogin">S\'inscrire à SimpleLogin</string>
+    <string name="sign_up_success">Vérifiez votre boîte de réception pour le code de vérification</string>
+    <string name="minimum_8_characters">Minimum 8 caractères</string>
     <!-- Verification -->
-    <string name="enter_otp">Enter OTP</string>
-    <string name="enter_activation_code">Enter activation code</string>
-    <string name="paste_and_verify">Paste &amp; verify</string>
-    <string name="code_found_from_the_clipboard">"%1$s" is found from the clipboard</string>
-    <string name="wrong_code_too_many_times">Wrong code too many times</string>
-    <string name="send_new_activation_code">We will send you a new activation code for "%1$s"</string>
-    <string name="check_new_verification_code">Check your inbox for new verification code</string>
+    <string name="enter_otp">Entrez le code OTP</string>
+    <string name="enter_activation_code">Entrez le code d\'activation</string>
+    <string name="paste_and_verify">Coller et vérifier</string>
+    <string name="code_found_from_the_clipboard">"%@" a été trouvé dans le presse-papiers</string>
+    <string name="wrong_code_too_many_times">Mauvais code saisi trop souvent</string>
+    <string name="send_new_activation_code">Nous vous enverrons un nouveau code d\'activation pour "%@".</string>
+    <string name="check_new_verification_code">Vérifiez votre boîte de réception pour le nouveau code de vérification</string>
     <!-- Time -->
-    <strind name="created_on">Created on</string>
-    <string name="just_now">just now</string>
-    <string name="minutes_ago">minutes ago</string>
-    <string name="day_ago">day ago</string>
-    <string name="days_ago">days ago</string>
-    <string name="week_ago">week ago</string>
-    <string name="weeks_ago">weeks ago</string>
-    <string name="month_ago">month ago</string>
-    <string name="months_ago">months ago</string>
-    <string name="year_ago">year ago</string>
-    <string name="years_ago">years ago</string>
+    <strind name="created_on">Créé le</string>
+    <string name="just_now">à l\'instant</string>
+    <string name="minutes_ago">il y a %@ minutes</string>
+    <string name="day_ago">il y a un jour</string>
+    <string name="days_ago">il y a %@ jours</string>
+    <string name="week_ago">il y a une semaine</string>
+    <string name="weeks_ago">il y a %@ semaines</string>
+    <string name="month_ago">il y a un mois</string>
+    <string name="months_ago">il y a %@ mois</string>
+    <string name="year_ago">il y a un an</string>
+    <string name="years_ago">il y a %@ ans</string>
```

Comment:
- Parameter must be added for plural strings
- Maybe replace `un` by `1` or by a parameter ?